shared/actions/search.js

```diff
@@ -31,50 +32,47 @@ type RawResult = {
   }
 }
 
-function parseFullName (rr: RawResult): string {
```

Comment:
I like this better, you're duplicating login on how to read full_name, this will give you the full_name in one spot, what's the reasoning to remove this?